Cargo.toml

```diff
@@ -1,6 +1,6 @@
 [package]
 name = "pic8259"
-version = "0.10.2"
```

Comment:
Please don't bump the version here. We do that seperately when we create a new release.

Reply:
Sorry about that. It was a mistake. Tried correcting it but I guess I did it wrongly.